pkg/bitcoin/local.go

```diff
@@ -0,0 +1,73 @@
+package bitcoin
+
+import "fmt"
+
+// LocalChain represents a local Bitcoin chain.
+type LocalChain struct {
```

Comment:
I think that should be placed inside `tbtc/maintainer` package. We rejected the idea of the single local chain implementation for `pkg/chain`'s interfaces in favor of per-application local chains (specific local implementations for `pkg/beacon`, `pkg/tbtc` and so on). I think we should take the same approach for `bitcoin.Chain` implementations. The per-application local chain approach is much more flexible and can be heavily customized in an app-specific way without creating a single monster implementation trying to satisfy all of the use cases. Of course, the per-app approach incurs some code duplication but this will be rather small so it's still worth it.

Reply:
Done in 2e2559f549d7b50daed63993c9a017edb8841713.